.github/workflows/lychee-push.yml

```diff
@@ -0,0 +1,13 @@
+on: push
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: lychee Link Checker
+        id: lychee
+        uses: lycheeverse/lychee-action@v1.0.6
+        with:
+          args: --exclude ".*.signalfx.com" --exclude "splunk.jfrog.io.*basearch" --exclude "localhost:.*" -v -n "*.md" "**/*.md"
```

Comment:
There's several download links and docs links worth verifying to signalfx.com. Looking at the list I wonder if you could get away with just excluding https://ingest.*.signalfx.com/

Reply:
appears to work so let's go with that for now